CMakeLists.txt

```diff
@@ -61,11 +61,20 @@ add_executable(CRender src/main.cpp
         src/ui/ui.h
         src/render/entities/components.h
         src/render/entities/registry.cpp
-        src/render/entities/registry.h src/render/draft/draft_renderer.cpp src/render/draft/draft_renderer.h src/render/timer.cpp src/render/timer.h src/util/logger.h src/util/logger.cpp src/util/numbers.h src/render/brdf.h src/util/denoise.h)
+        src/render/entities/registry.h 
+        src/render/draft/draft_renderer.cpp 
+        src/render/draft/draft_renderer.h 
+        src/render/timer.cpp 
+        src/render/timer.h 
+        src/util/logger.h 
+        src/util/logger.cpp 
+        src/util/numbers.h 
+        src/render/brdf.h 
+        src/util/denoise.h)
 
 target_include_directories(CRender PRIVATE src)
 target_include_directories(CRender PRIVATE external)
 
-target_link_libraries(CRender glfw fmt glm embree OpenImageDenoise)
+target_link_libraries(CRender glfw fmt glm embree OpenImageDenoise stdc++fs)
```

Comment:
I don't think we need to do this. (This seems to be needed to be done when there's an error somewhere else)

Reply:
Unless someone is using a compiler from the 1800's (old one) we won't need it, might as well remove it then.